src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import { shell, ShellResult } from '../../../shell';
+
+export async function startMinikube() {
+    shell.exec('minikube start').then((result: ShellResult) => {
+        if (result.code == 0) {
```

Comment:
In before @bnookala - prefer `===` and `!==` unless we really consciously want evil type coercion.  (There's a couple of other places this comes up in this PR.)

Reply:
fixed.